django_cloud_tasks/tasks/task.py

```diff
@@ -232,9 +233,9 @@ def _calculate_delay_in_seconds(eta: int | timedelta | datetime) -> int:
         if isinstance(eta, int):
             return eta
         elif isinstance(eta, timedelta):
-            return int(eta.total_seconds())
```

Comment:
I'm assuming this is here because `delay` has a resolution of seconds. If so, there are multiple ways to think about it:

1. If I ask for a delay of 0.4 seconds, I should not delay. If I ask for a delay of 0.6 seconds, I delay 1 second (use closest approximation)
2. Delay for "at least" the requested delay (this is what this PR implements)
3. Delay for "at most" the requested delay (this is what the previous implementation did)

From the point of view of `django_cloud_tasks` (which is a lib separate from the Nilo ecossystem), it is not obvious to me why approach #2 is the "right" one. If anything I'd pick approach #1 and use "round" here.

The caller can always choose to interpret this differently by passing the eta in seconds directly and not relying on inferring it from a timedelta or a timestamp.